settings.gradle

```diff
@@ -0,0 +1,4 @@
+rootProject.name = 'volley'
+include 'core'
+include 'cronet'
+include 'testing'
```

Comment:
nit: add an empty line at the end of the file to suppress the Github warning

Reply:
done